packages/amplify-category-function/src/__tests__/provider-utils/awscloudformation/utils/environmentVariablesHelper.test.ts

```diff
@@ -1,15 +1,13 @@
-import {
-  stateManager, JSONUtilities, $TSContext, pathManager,
-} from 'amplify-cli-core';
+import { stateManager, JSONUtilities, $TSContext, pathManager } from 'amplify-cli-core';
 import { prompter } from 'amplify-prompts';
-import {
-  ensureEnvironmentVariableValues,
-  getStoredEnvironmentVariables,
-  saveEnvironmentVariables,
-} from '../../../../provider-utils/awscloudformation/utils/environmentVariablesHelper';
+import * as m from '../../../../provider-utils/awscloudformation/utils/environmentVariablesHelper';
+import inquirer from 'inquirer';
+import * as uuid from 'uuid';
 
 jest.mock('amplify-cli-core');
 jest.mock('amplify-prompts');
+jest.mock('inquirer');
```

Comment:
also I though we were moving away from inquirer to prompter...

Reply:
yep, should have been removed. Doing that now